src/QualificationRunner.Core/Assets/Resources.cs

```diff
@@ -23,7 +23,7 @@ public static class Errors
 
       public static string ReferencedProjectNotDefinedInQualificationFile(string project) => $"Referenced project '{project}' is not defined in qualification file";
 
-      public static string ProjectConfigurationNotValid(string project, string logFile) => $"Project configuration for '{project}' is invalid. Please check log file at '{logFile}' for details.";
+      public static string ProjectConfigurationNotValid(string project, string logFile) => $"Project configuration for '{project}' is invalid. Please check one of the log files at '{logFile}' for details.";

```

Comment:
I think here we only want the one project specific log file, not the main one

Reply:
done